rcl_lifecycle/QUALITY_DECLARATION.md

```diff
@@ -88,14 +88,30 @@ The results of the test can be found [here](https://ci.ros2.org/view/nightly/job
 
 `rcl_lifecycle` has feature tests, which test for proper node state transitions.
 The tests are located in the [test](test) subdirectory.
+New features are required to have tests before being added.
+Currently nightly test results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/rcl_lifecycle/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rcl_lifecycle/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/rcl_lifecycle/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/rcl_lifecycle/)
+
 
 ### Public API Testing [4.ii]
 
-Much of the API in `rcl_lifecycle` is tested in the aforementioned feature tests, but it is not tested explicitly.
+Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [86%](https://ci.ros2.org/job/ci_linux_coverage/lastBuild/cobertura/src_ros2_rcl_rcl_lifecycle_src/).
```

Comment:
All of the API is tested. That's separate from line coverage. The only parts that aren't covered right now are private functions pretty much.